doc/source/user_reference/agenda.rst

```diff
@@ -0,0 +1,99 @@
+should be enabled and disabled and also to provide any relevant configuration.
+The ``"augmentation"`` entry, if present, should be a list of augmentations that
+should be enabled (or if prefixed with a ``~``, disabled). In order to supply
+configuration for an augmentation it's name should be provided as an entry in
+this section and should contain a mapping of configuration options to their
+desired settings. If configuration is supplied for an augmentation that is not
+enabled then it will simply be ignored. This allows for augmentations to be
+temporarily removed without also having to remove their configuration, or to
+provide a set of defaults for an augmentation which can then be overridden.
+
+.. note:: While augmentations can be enabled and disabled on a per workload
+          basis, they cannot yet be re-configured part way through a run and the
+          configuration provided as part of an agenda config section or separate
+          config file will be used for all jobs in a WA run.
+
+
+Workloads
+^^^^^^^^^
+
+Here you can specify a list of global workloads to be ran. If you wish to run a
+workload with all default values then you can specify the workload name directly
+as an entry, otherwise a dict mapping should be provided. Any settings provided
+here will be the most specific and therefore override any other more generalised
+configuration for that particular workload spec. The valid entries are as
+follows:
+
+- :confval:`workload_name` (Mandatory) - The name of the workload to be ran
+- :confval:`iterations` - Specify how many iterations the workload should be ran
+- :confval:`label` - Similar to IDs but do not have the uniqueness restriction.
+  If specified, labels will be used by some output processors instead of (or in
+  addition to) the workload name. For example, the csv output processor will put
+  the label in the "workload" column of the CSV file.
+- :confval:`augmentations` - The instruments and output processors to enable (or
+  disabled using a ~) during this workload.
+- :confval:`classifiers` Classifiers allow you to tag metrics from this workload
+  spec which are often used to help identify what runtime parameters were used
+  when post processing results.
+- :confval:`workload_parameters` - Any parameters to configure that particular
+  workload in a dict form.
+
+      .. note:: You can see available parameters for a given workload with the
+                :ref:`show command <show-command>`.
+
+- :confval:`runtime_parameters` - A dict mapping of any runtime parameters that
+  should be set for the device for that particular workload. For available
+  parameters please see :ref:`runtime parameters <runtime-parameters>`.
+
+     .. note:: Unless specified elsewhere these configurations will not be
+               undone once the workload has finished. I.e. if the frequency of a
+               core is changed it will remain at that frequency until otherwise
+               changed.
+
+
```

Comment:
Should have a note here saying that there is also an even shorter `params` alias, however that gets interpreted differently depending on whether it is used in workload entry, in which case it will be interpreted as `workload_params`, or at global `config` or `section` (see below) level, in which case it will be interpreted as `runtime_params`.